modules/querier/querier_query_range.go

```diff
@@ -116,7 +116,12 @@ func (q *Querier) queryBlock(ctx context.Context, req *tempopb.QueryRangeRequest
 		timeOverlapCutoff = v
 	}
 
-	eval, err := traceql.NewEngine().CompileMetricsQueryRange(req, false, timeOverlapCutoff, unsafe)
+	exemplarsEnabled := req.Exemplars
```

Comment:
Same comment here as the local-blocks processor.  The querier isn't using `exemplarsEnabled` directly.